src/main/java/edu/hawaii/its/groupings/controller/ErrorRestController.java

```diff
@@ -29,4 +27,12 @@ public ResponseEntity<Void> feedbackError(@RequestBody Map<String, String> body,
         session.setAttribute("feedback", new Feedback(exceptionMessage));
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
-}
+
+    @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping(value = "/api/groupings/testing/exception")
```

Comment:
Sorry final thing, could you change the URI to just /testing/exception? Make sure to update grouping.service.js throwException(). I think it's weird to have /api/groupings since we use this prefix for the GroupingsRestController.